mlc_llm/quantization/quantization.py

```diff
@@ -107,7 +107,14 @@ class QuantizationScheme:
     embedding_table: QuantizationSpec
     final_fc_weight: QuantizationSpec
     others: QuantizationSpec
-
+    pre_quantized: bool
+    
+    _base_model_prefix: str
+    _layers_block_name: str
+    _outside_layer_modules: List[str]
+    _inside_layer_modules: List[List[str]]
+    _load_quantized_params_func: Optional[Callable] 
```

Comment:
Could you add some brief docstring for these fields?

Reply:
I have put some of theses params' docs, checkout this https://github.com/LeiWang1999/mlc-llm/blob/a96d6ce4ef479c849ea2abcfec3935e8d09145ae/mlc_llm/quantization/quantization.py#L112-L143 , don't know if the format can be improved.

Reply:
Thanks! It’s fine and no worries. I can take a look and update them later on my side.